### PR TITLE
Peregrine page > syntax fixes

### PR DESCRIPTION
**Pull request by rafaelstz**
## Description

In this PR I fixed 2 issues related to the text syntax for the Peregrine page.

## Checklist:
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have linked an issue to this PR.
- [ ] I have indicated the change type and relevant package(s).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] All CI checks are green (linting, build/deploy, etc).
- [ ] At least one core contributor has approved this PR.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=1031) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento-research/pwa-studio?pullRequest=1031) before we can accept your contribution.<br/>

File: pwa-devdocs/src/peregrine/index.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 title: Peregrine
 ---
 
-The Peregrine project is a collection of UI components for Magento PWA projects.
+The Peregrine project is a collection of UI Components for Magento PWA projects.
```
Comment (jcalcaben) [inline pwa-devdocs/src/peregrine/index.md]: The lowercase form of 'components' is the correct form in this context.
They are nothing more than components for UI.

Using 'UI Components' imply an official label for these components, which is not desired.

File: pwa-devdocs/src/peregrine/index.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
 Use, extend, or remix these components to create a unique Magento PWA storefront.
 
-Visit the [`peregrine`] repository to contribute to this project.
+Visit the [`Peregrine`] repository to contribute to this project.
```
Comment (jcalcaben) [inline pwa-devdocs/src/peregrine/index.md]: **Peregrine** is the name of the project.
`peregrine` is the name of the repository.

This sentence refers to the repository, so the original `peregrine` form is the correct one.

File: pwa-devdocs/src/peregrine/index.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
...
 
-[`peregrine`]: https://github.com/magento-research/pwa-studio/tree/master/packages/peregrine
\ No newline at end of file
+[`Peregrine`]: https://github.com/magento-research/pwa-studio/tree/master/packages/peregrine
```
Comment (jcalcaben) [inline pwa-devdocs/src/peregrine/index.md]: See previous comment.

**Pull request closed without merging**